### DISCUSSION TITLE
Filtering null [May 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2001] [Discord Threads]
> 🙋‍♂️ **David (he/him)**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### there is an operator called `is null`. You should use this for checking null columns  
#  
> 🙋‍♂️ **David (he/him)**  
#### that's the first thing I tried :)  
#  
> 👨‍🚀 **wingkwong**  
#### please create an issue in github then.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/969700317946077264)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
